### ISSUE TITLE
Ability to list merge requests assigned to a user

### ISSUE THREAD

**Issue by profclems** [profclems/glab#167]
**Is your feature request related to a problem? Please describe.**
There should be additional flags on the `glab mr list` command to list merge request assigned to current user or a specific user

**Describe the solution you'd like**
Add --mine and --assignee flags to `glab mr list`

To get list of MRs for the current user:
```sh
glab mr list --mine
```
To get the list of MRs for a user with username
```sh
glab mr list --assignee=<username>
```
Should allow multiple assignees listing
```sh
glab mr list --assignee=user1,user2
```

**Comment by zemzale** [comment]
Since I had listing only mine MR done I figured I could implement the `glab mr list --assignee=<username>`, part too.

I came up with few problems. GitLab allows filtering only by single [assignee ID ](https://docs.gitlab.com/ce/api/merge_requests.html#list-project-merge-requests). That might not be a problem since we could do multiple request, one per user, and then display.

Before we can make the request for every user ID we need to search for the users by their names. That is done through the [UsersService](https://docs.gitlab.com/ce/api/users.html#list-users) using search. There are few things that we need to consider : 
- I assume we take in account only exact matches, since the search returns values that are not exact matches too.
- Do we display anything about Users that don't have any MRs assigned to them, or just omit them?

Just wanted to clarify these things before start implementing them, or just get some advice about my idea how to approach it since this might not be the optimal solution.
